Makefile

```diff
@@ -259,7 +259,7 @@ kafka: deploy-kafka-operator
 	@kubectl create namespace $(KAFKA_NAMESPACE) 2>&1 | grep -v "already exists" || true
 	@curl --fail --location $(KAFKA_EXAMPLE) --output deploy/test/kafka-example.yaml --create-dirs
 	@sed -i 's/size: 100Gi/size: 10Gi/g' deploy/test/kafka-example.yaml
-	@kubectl -n $(KAFKA_NAMESPACE) apply --dry-run=client -f deploy/test/kafka-example.yaml
+	@kubectl -n $(KAFKA_NAMESPACE) apply --dry-run=true -f deploy/test/kafka-example.yaml
```

Comment:
@kevinearls I think you added this flag in the past. Not sure where this option is parsed, but it currently fails with my client/server combination. I got it working by setting it to `client`, but wanted to check with you whether this is indeed the correct value.

For reference, here are the versions I'm using:

```
Client Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.0-4-g38212b5", GitCommit:"d038424d6d4f1cc39ad586ac0d36dac3a7a37ceb", GitTreeState:"clean", BuildDate:"2020-06-16T03:29:46Z", GoVersion:"go1.13.4", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.3", GitCommit:"2e7996e3e2712684bc73f0dec0200d64eec7fe40", GitTreeState:"clean", BuildDate:"2020-05-20T12:43:34Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
```